modules/chat/src/main/Line.scala

```diff
@@ -23,6 +25,12 @@ case class UserLine(
 
   def isLichess = userId.is(UserId.lichess)
 
+object UserLine:
+  private[chat] given BSONHandler[UserLine] = BSONStringHandler.as[UserLine](
```

Comment:
Split this from `Line` object, otherwise it'll be `BSONHandler[UserLine]` and `BSONHandler[Line]` are ambiguous.